bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/config/config.xml

```diff
@@ -11,6 +11,11 @@
 			<label>API Key</label>
 			<description>API key to access the OpenWeatherMap API.</description>
 		</parameter>
+		<parameter name="apiVersion" type="text" required="true">
```

Comment:
Please remove the "required" Attribute. 

Please also update the default i18n property file (openweathermap.properties).

Reply:
It would be a good decision to add an options list here as there are only two configuration values possible.will be easier for the user to handle.

Reply:
Good suggestion

Reply:
Have updated the i18n, removed required, and changed to an options list